### PR TITLE
Update pyega version from 4.0.5 to 5.0.2

### PR DESCRIPTION
**Pull request by JasperO98**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by bgruening** [comment]
Thanks @JasperO98. Can you please check the generated outputs? I'm happy to merge as is, but we need to sanity check - not that we are generating empty files.

File: tools/pyega3/pyega3.xml [hunk 1]
```diff
@@ -213,8 +213,8 @@
             <param name="start" value="0"/>
             <param name="end" value="10000"/>
             <output_collection name="downloaded_file_collection" count="2">
-                <element name="NA19239_genomic_range_1_0_10000" md5="bcdcf18846233cbe5cc8afd95168552c" />
```
[the remark this answers is not in the document]
Comment (JasperO98) [reply tools/pyega3/pyega3.xml:216]: I manually checked the bam files and they are both not empty. What would be a good way to verify this in the test? I see `<has_size value="X" delta="Y"/>` being used for `.h5` files do you think that would be sufficient?

[the remark this answers is not in the document]
Comment (bgruening) [reply tools/pyega3/pyega3.xml:216]: Yeah, has_size can be good.
BAM is also special as it gets unpacked imho. So we maybe can also use the text asserts, like "contain", "has_line" etc... but has_size is a very good start.

**Comment by JasperO98** [comment]
Although the BAMs are not empty I get a samtools error: `samtools view: error reading file`. So something is going wrong with the download, but that's on becayse of PyEGA3 not Galaxy itself...

**Comment by bgruening** [comment]
Thanks @JasperO98!

**Pull request merged**